tests/unit/calendar/calendar_core.js

```diff
@@ -139,7 +139,7 @@ asyncTest( "keyboard handling", function() {
 			.focusGrid( element )
 			.simulate( "keydown", { keyCode: $.ui.keyCode.LEFT } );
 		setTimeout(function() {
-			$( ":focus" ).simulate( "keydown", { keyCode: $.ui.keyCode.ENTER } );
+			$( document.activeElement ).simulate( "keydown", { keyCode: $.ui.keyCode.ENTER } );
```

Comment:
Alright, that's fine by me. Can you make a note on the list? And go ahead and merge this.